core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -853,6 +857,18 @@ public void install() {
                     LOGGER.log(Level.WARNING, "Dependent install of " + dep.name + " for plugin " + name + " already added, skipping");
                 }
             }
+            PluginWrapper pw = getInstalled();
+            if(pw != null) { // JENKINS-34494 - check for this plugin being disabled
+                Future<UpdateCenterJob> enableJob = null;
+                if(!pw.isEnabled()) {
```

Comment:
Is this the case when a newly installed plugin has, for whatever reason, a `.disabled` file on disk? Or how can this happen?